agent/battle/vision/prompts.py

```
@@ -0,0 +1,27 @@
+scene 只能是 unknown、main_battle、command_selection、skill_target_selection、order_change、victory、defeat、dialog。
+卡色只能是 B、A、Q。不要输出 Markdown、解释文字或 JSON 之外的内容。"""
+
+
+def build_user_prompt(request: VisionRequest) -> str:
+    context = {
+        "turn_index": request.turn_index,
+        "requested_fields": list(request.requested_fields),
+        "recent_actions": list(request.recent_actions),
+        "mfw_state": repr(request.state) if request.state is not None else None,
+    }
+    return "请分析附带截图，只返回符合 schema 的 JSON。上下文：\n" + json.dumps(
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)